be/src/storage/olap_meta_reader.cpp

```diff
@@ -95,7 +95,7 @@ Status OlapMetaReader::_build_collect_context(const OlapMetaReaderParams& read_p
         }
         _has_count_agg |= (collect_field == "count");
     }
-    _collect_context.seg_collecter_params.tablet_schema = read_params.tablet->tablet_schema();
+    _collect_context.seg_collecter_params.tablet_schema = read_params.tablet_schema;
```

Comment:
1. If FE doesn't record this infos, how it show the `bloom filter` when `show create table`?
2. If the bloom filter has this issue, does the bitmap index has this issue too?

Reply:
> 1. If FE doesn't record this infos, how it show the `bloom filter` when `show create table`?
> 2. If the bloom filter has this issue, does the bitmap index has this issue too?

1. The attributes of most columns are primarily stored within the columns themselves, except for the property of whether to create a Bloom filter. The FE maintains the names of columns with Bloom filters. As a result, when you execute a "show create table" command, you can see this information. However, the FE loses the Bloom filter property when transmitting column information to the BE(in function `Column.toThrift()`)